components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -39,11 +39,13 @@ interface KVCondition {
   styleUrls: ['./project-rules.component.scss']
 })
 export class ProjectRulesComponent implements OnInit, OnDestroy {
-  public project: Project = <Project>{};
   public ruleId: string;
   public ruleForm: FormGroup;
+
+  // FIXME: either make properties optional in interface, or provide them on initialization:
```

Comment:
what's the issue here?

Reply:
The interfaces defined for `Rule` and `Project` don't allow for `{}`; because `{}` is lacking non-optional properties. So, we use force to make the compiler accept that, via `<Project>{}`. That's a footgun. 😄